src/App.vue

```diff
@@ -4,6 +4,7 @@
     <ToDos v-bind:todoEntries="todoEntries" @delete-todo-event="deleteTodDoItem" />
     <AddToDoButton @add-todo-event="addToDoItem" />
   </div>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```

Comment:
Diese Variante ist ist korrekt und funktioniert. Im HTML kann man mit dem `<link />` oder `<script>` tag code auch von anderen Webseiten laden. Solche Webseiten heißen CDN (Content Delivery Network). 

Da du aber oben boostrap auch schon installiert hast, kannst du es aber auch von deinem eigenen Projekt inkludieren und brauchst nicht über das internet zu gehen. 

lösche diese Zeile raus und füge folgendes in der `main.js` datei auf Zeile 3 hinzu:

```
import 'bootstrap/dist/css/bootstrap.min.css';
import 'bootstrap';
```

auf diese Weise kannst du Bootstrap direkt von deinem Computer aus einbinden. 
Dies würdest du auch bei einem echten Projekt so machen, da es viel schneller ist für deine User.